package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "colette",
-  "version": "1.16.0",
```

Comment:
Weird, version changed without diff in package-lock.json

Reply:
I guess forgot to run `npm install`

Reply:
@nredoulez and I found it was an old npm version running `npm version minor` command